js/signin/constants.js

```diff
@@ -0,0 +1,12 @@
+const VALIDATION_MESSAGES = {
```

Comment:
상수 파일에 다른 상수들이 추가될 수 있으므로, export default 보다 named Export를 쓰는게 더 확장성이 좋아보여요.
```suggestion
export const VALIDATION_MESSAGES = {
```
(밑의 export default 는 삭제)